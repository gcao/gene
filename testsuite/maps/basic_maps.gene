#!/usr/bin/env gene run
# TEST: Basic map operations

# Map creation
(var person {
  "name" "Alice"
  "age" 30
})
(print person)

# Map access
(print (person "name"))
(print (person "age"))