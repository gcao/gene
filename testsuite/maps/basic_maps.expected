{"name" "Alice" "age" 30}
Alice
30